examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/GenerateUtil.java

```diff
@@ -0,0 +1,131 @@
+     * @param outputPath output path
+     * @throws IOException IO exception
+     * @throws TemplateException template exception
+     */
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    public static void generateDirs(final Configuration templateConfig, final Map<String, String> dataModel, final Collection<String> paths, final String outputPath) throws IOException, TemplateException {
+        if (null == paths || 0 == paths.size()) {
+            new File(generatePath(templateConfig, dataModel, outputPath)).mkdirs();
+            return;
+        }
+        for (String each : paths) {
+            new File(generatePath(templateConfig, dataModel, outputPath + "/" + each)).mkdirs();
+        }
+    }
+    
+    /**
+     * Generate file.
+     * 
+     * @param templateConfig template config
+     * @param dataModel data model
+     * @param path paths
+     * @throws IOException IO exception
+     * @throws TemplateException template exception
+     */
+    public static void generateFile(final Configuration templateConfig, final Map<String, String> dataModel, final Map<String, String> templateMap, final String path) throws IOException, TemplateException {
+        String outputPath = generatePath(templateConfig, dataModel, path);
+        for (Map.Entry<String, String> entry : templateMap.entrySet()) {
+            processFile(templateConfig, dataModel, entry.getKey(), outputPath + "/" + entry.getValue());
+        }
+    }
+    
+    /**
+     * Generate path.
+     *
+     * @param templateConfig template config
+     * @param model model
+     * @param relativePath relative path
+     * @return Path generated from the model
+     * @throws IOException IO exception
+     * @throws TemplateException template exception
+     */
+    public static String generatePath(final Configuration templateConfig, final Object model, final String relativePath) throws IOException, TemplateException {
+        try (StringWriter result = new StringWriter()) {
+            new Template("path", relativePath, templateConfig).process(model, result);
+            return result.toString();
+        }
+    }
+    
+    /**
+     * Generate file.
+     *
+     * @param templateConfig template config
+     * @param model data model
+     * @param templateFile template file
+     * @param outputFile output file
+     * @throws IOException IO exception
+     * @throws TemplateException template exception
+     */
+    public static void processFile(final Configuration templateConfig, final Object model, final String templateFile, final String outputFile) throws IOException, TemplateException {
+        try (Writer writer = new FileWriter(outputFile)) {
+            templateConfig.getTemplate(templateFile).process(model, writer);
+        }
+    }
+    
+    /**
+     * Combination generator.
+     * 
+     * @param combs combs
+     * @return All combination
+     */
+    public static Collection<String> generateCombination(String[] combs) {
```

Comment:
Please do not use abbreviation in Java Doc, it is better to use `combinations` to instead of `combs`

Reply:
done.

Reply:
The `String[] combs` has not changed yet

Reply:
my fault,done.